src/Native/Console/NativeCom.js

```diff
@@ -17,8 +17,13 @@ Elm.Native.Console.NativeCom.make = function(localRuntime) {
     var Utils = Elm.Native.Utils.make(localRuntime);
 
     /* Node.js imports */
-    var fs = require('fs');
-    var stdin = process.stdin;
+    try {
+        var fs = require('fs');
+        var stdin = process.stdin;
+        var in_node = true;
+    } catch (reference_error) {
+        var in_node = false;
```

Comment:
(Sorry for commenting from non-contributor)

How about this way to check:

```js
var in_node = typeof module !== 'undefined' && module.exports
```

Ref: http://stackoverflow.com/questions/4224606/how-to-check-whether-a-script-is-running-under-node-js

Then, maybe sneakCase is preferable. `inNode`